svsim/src/main/scala/Backend.scala

```diff
@@ -22,10 +22,10 @@ object CommonCompilationSettings {
     def apply(name: String) = new VerilogPreprocessorDefine(name, None)
   }
   case class VerilogPreprocessorDefine private (name: String, value: Option[String]) {
-    private[svsim] def toCommandlineArgument: String = {
+    final def toCommandlineArgument(backend: Backend): String = {
```

Comment:
Did you mean to make this public?

Reply:
Yeah, there was an internal duplicate use of this. It seems less bad to make this public to avoid the duplication internally. I'll land this with this change in place and will re-privatize it once the internal usage is converted to ChiselSim.